apps/src/code-studio/pd/application_dashboard/quick_view_table.jsx

```diff
@@ -54,16 +54,13 @@ export default class QuickViewTable extends React.Component {
       },
       cell: {
         format: (status) => {
-          if (status === 'move_to_interview') {
-            return 'Move to Interview';
-          }
           return _.upperFirst(status);
         },
         transforms: [
           (status) => ({
             style: {
-              backgroundColor: this.getViewColor(status),
-              color: this.getTextColor(status),
+              backgroundColor: StatusColor[status],
+              color: StatusTextColor[status],
```

Comment:
I didn't notice this earlier, but this should be `style: styles[status]` or something similar with all the style defined in a `styles` object according to our [styleguide](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#react). You should be able to rework the exported styles above so all styles (color, backgroundColor, and padding) are in a single object for each status. 

You could keep the common `padding` on its own and do something like `styles: {...styles.statusCellCommon, ...styles.statusCell[status]}`